sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/PullupCorrelatedPredicatesSuite.scala

```diff
@@ -27,6 +27,8 @@ import org.apache.spark.sql.catalyst.rules.RuleExecutor
 class PullupCorrelatedPredicatesSuite extends PlanTest {
 
   object Optimize extends RuleExecutor[LogicalPlan] {
+    override protected val blacklistedOnceBatches = Set("PullupCorrelatedPredicates")
```

Comment:
shall we also change `Once` to `FixedPoint(1)` here instead of adding blacklist?

Reply:
I don't think so cuz currently this rule generates wrong plans if run multiple times. See #25164